sdk/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "beta9"
-version = "0.1.35"
+version = "0.1.37"
```

Comment:
0.1.36 already exists. Just wasn't committed which is why I'm going straight to 0.1.37.